CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to Zonemaster::Backend
+
+Contribution to this repository is welcome. Contribution can be either an issue
+report or a code or a documentation update. Also see the information in the
+[main README][Zonemaster/Zonemaster README] in the main Zonemaster respository.
+
+## Issue
+
+First search for a similar issue in the [issues list]. If a relevant issue is
+found, add your information as a comment. If no relevant issue is found, create a
+[new][create issue] issue. Give as many details as you have and describe, if
+possible, how the issue can be reproduced.
+
+## Pull request
+
+If you have a solution for an issue, or a suggestion for an improvment, please
+first search for a pull request for the same issue among open [pull requests].
+If nothing relevant is found or you want to contribute with a different solution,
+create a [new][create pull request] pull request, which assumes that you have
+your proposal in a fork repository.
```

Comment:
Maybe integrate some wording along these lines?

> If there are any issues or open pull requests that are related to your change please mention them in the pull request description and say something about how they relate.

Reply:
Maybe such a suggestion would discourage people from contributing if they feel the paperwork is taxing?
Maybe we should keep the instructions as lightweight as possible and only expand them to solve real problems, and also to increase the likelihood that people actually base their pull requests on the develop branch rather than the master branch.

Reply:
I am not sure what you want.